GitUI/UserControls/FileStatusList.Designer.cs

```diff
@@ -104,8 +104,8 @@ private void InitializeComponent()
             FilterComboBox.Location = new Point(0, 0);
             FilterComboBox.Margin = new Padding(0, 0, 0, 1);
```

Comment:
This unnecessary gap has not come into effect before this PR.

```suggestion
            FilterComboBox.Margin = new Padding(0);
```

Reply:
removed